sandbox/cgroup/freezer.go

```diff
@@ -82,36 +86,44 @@ var ThawSnapProcesses = thawSnapProcessesImplV1
 // freezeSnapProcessesImplV1 freezes all the processes originating from the given snap.
 // Processes are frozen regardless of which particular snap application they
 // originate from.
-func freezeSnapProcessesImplV1(snapName string) error {
+var freezeSnapProcessesImplV1 = func(ctx context.Context, snapName string) error {
 	fname := filepath.Join(freezerCgroupV1Dir, fmt.Sprintf("snap.%s", snapName), "freezer.state")
-	if err := os.WriteFile(fname, []byte("FROZEN"), 0644); err != nil && os.IsNotExist(err) {
+	if err := os.WriteFile(fname, []byte("FROZEN"), 0644); err != nil && errors.Is(err, fs.ErrNotExist) {
```

Comment:
Given that `errors.Is` checks the error, I think it's fine to remove `err != nil` condition (here and elsewhere).